pkg/kubelet/config/file.go

```diff
@@ -173,32 +173,32 @@ func (s *sourceFile) extractFromDir(name string) ([]*v1.Pod, error) {
 	for _, path := range dirents {
 		statInfo, err := os.Stat(path)
 		if err != nil {
-			klog.Errorf("Can't get metadata for %q: %v", path, err)
+			klog.ErrorS(err, "Could not get metadata", "path", path)
 			continue
 		}
 
 		switch {
 		case statInfo.Mode().IsDir():
-			klog.Errorf("Not recursing into manifest path %q", path)
+			klog.ErrorS(nil, "Not recursing into manifest path", "path", path)
 		case statInfo.Mode().IsRegular():
 			pod, err := s.extractFromFile(path)
 			if err != nil {
 				if !os.IsNotExist(err) {
-					klog.Errorf("Can't process manifest file %q: %v", path, err)
+					klog.ErrorS(err, "Could not process manifest file", "path", path)
 				}
 			} else {
 				pods = append(pods, pod)
 			}
 		default:
-			klog.Errorf("Manifest path %q is not a directory or file: %v", path, statInfo.Mode())
+			klog.ErrorS(nil, "Manifest path is not a directory or file", "path", path, "mode", statInfo.Mode())
```

Comment:
Not sure we should be using `ErrorS` here as there isn't an error. Perhaps `InfoS`? @serathius what do you think?

Reply:
From migration guide
`Calling ErrorS with nil as error is semi-acceptable if there is error condition that deserves a stack trace at this origin point. For expected errors (errors that can happen during routine operations) please consider using klog.InfoS and pass error in err key instead.`
I don't think this is expected situation and using `ErrorS` with nil is ok here.